books/free-programming-books-langs.md

```diff
@@ -538,6 +538,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Joint Strike Fighter, C++ Coding Standards](http://www.stroustrup.com/JSF-AV-rules.pdf) - Bjarne Stroustrup (PDF)
 * [Learn C++ Programming Language](http://www.tutorialspoint.com/cplusplus/cpp_tutorial.pdf) - Tutorials Point (PDF)
 * [LearnCpp.com](https://www.learncpp.com) (HTML)
+* [The Complete Reference C++,Herbert Schildt](https://www.micc.unifi.it/bertini/download/labtecinf/C%2B%2B%20A%20Beginner's%20Guide%202nd%20Edition%20(2003).pdf) - Second Edition
```

Comment:
You must follow the rule described [here](https://github.com/vhf/remark-lint-books-links)
For example
```test
[Another Awesome Book](http://example.com/book.pdf) - John Doe (PDF)
```
So for your case, the proper way to do this is
```text
[The Complete Reference C++ (second edition)](https://www.micc.unifi.it/bertini/download/labtecinf/C%2B%2B%20A%20Beginner's%20Guide%202nd%20Edition%20(2003).pdf) - Herbert Schildt (PDF)
```